package.json

```diff
@@ -14,12 +14,13 @@
     "nohoist": []
   },
   "scripts": {
-    "build": "lerna run build --stream --prefix --npm-client yarn",
+    "build": "lerna run build --stream --prefix --npm-client yarn && cd packages/icons-angular && yarn build:full",
```

Comment:
modifying the primary build script for a single package is not a good idea, if we do this for every package it would be insane.

Reply:
Totally open to suggestions 👍 How would you recommend we approach this? Keep in mind we've hit both the CircleCI and node memory limit at various points. The node limit seems to be under control, but I worry that in the lerna streaming build we may exceed the CircleCI limit.

I'll push a test commit to see if we run into any issues.

Reply:
Yep, so seems like under the streaming build we do indeed hit the CircleCi resource limit: `@carbon/icons-angular: error Command failed with exit code 137.` where `137` usually indicates a resource error (i.e. out of memory).

As much as I appreciate that modifying the primary build script isn't ideal, we do have a fairly exceptional circumstance on our hands - unless there's some other way to run this particular build in isolation.

Reply:
I think the preference would be to pull things out of the monorepo if we have a situation like this versus slow down the whole, if that makes sense.

Definitely agreed, this is a crazy exceptional situation that we're still dealing with on the react side. Just so many assets to ship.

Reply:
> I think the preference would be to pull things out of the monorepo if we have a situation like this versus slow down the whole, if that makes sense.

Sure ... My only concern would be with syncing updates - I don't want this package to lag behind.

As an alternative we can look at adding a Github action for this package and excluding it from the local build (Don't get me wrong, the build times are kinda excessive) ... At least there we can be sure we're synced up.

Another alternative could be pulling all the icon packages out - given that the react and vue build will also continue to grow.

I'll take a look at adding an action for this ... but let me know if you have any other ideas 👍